src/binder_network.c

```diff
@@ -1372,6 +1374,21 @@ binder_network_can_set_initial_attach_apn(
         !self->set_data_profiles_req;
 }
 
+static
+gboolean
+binder_attach_apn_retry(
+    RadioRequest* req,
+    RADIO_TX_STATUS status,
+    RADIO_RESP resp,
+    RADIO_ERROR error,
+    const GBinderReader* args,
+    void* user_data)
+{
+    BinderNetworkObject* self = THIS (user_data);
+
+    return !gutil_ints_contains (self->ignore_apn_errors, error);
```

Comment:
Minor nitpick - space before the opening bracket (here and at least in one more place) is slightly out of style

Reply:
Fixed :)